### DISCUSSION TITLE
How is the project private?

### DISCUSSION THREAD

**Discussion by wiktordaniec** [logancyang/obsidian-copilot#905] [General]
I am a little confused by how you market the product. You are saying it's privacy first and focus on local storage, but you are using cloud providers.

Even if it's stored locally, even if you index it locally, you must send the content of the note to OpenAI or Claude servers. It completely misses the point of privacy. I feel like you are misleading the users.

**Comment by logancyang** [comment]
@andrzejZdobywca You can use LOCAL models.

**Comment by j-underdown** [comment]
@logancyang where can I find the privacy policy? I've seen it written that it is "on the website" but I cannot find it on the website. Can you please provide a direct link?

**Comment by piotrlo** [comment]
https://github.com/logancyang/obsidian-copilot/blob/master/local_copilot.md

**Comment by motion2082** [comment]
Agree with OP, it's a little misleading as you think only the embedding model is required to run locally (privately) but what you're actually doing is using the embed model to index your Vault then the External Open AI API request to send that out to Open AI to do the heavy lifting. 

I'm experimenting with Local LLM via LM Studio at running

Chat 
- Llama 3.2 3B Instuct
- Deepseek R1 1.5B 

Embed
- deepseek-r1-distill-qwen-1.5b

![image](https://github.com/user-attachments/assets/a5ea2fde-e3fc-43a0-9351-be0ec2502b83)

See
https://youtu.be/hOaSO_e7MYs?si=ROZF680n0BloHhb7
